extensions/deployers/runtime_zip_deploy.py

```diff
@@ -3,6 +3,23 @@
 import zipfile
 
 
+def _get_files_from_folder(search_dir, output_folder):
+    # recusively copy all .dlls and .exes from the search_dir to the output_folder
+    if not os.path.isdir(search_dir):
+        return []
+    files = []
+    for f in os.listdir(search_dir):
```

Comment:
I wonder if using os.walkdir is a better approach here, but I like the current minimal diff nevertheless